### PR TITLE
refactor: simplify the conversion from int to time.Duration as suggested by the golang offical doc.

### PR DESCRIPTION
**Pull request by wzy9607**
见 https://pkg.go.dev/time#pkg-constants

```
To convert an integer number of units to a Duration, multiply:

seconds := 10
fmt.Print(time.Duration(seconds)*time.Second) // prints 10s

```

### CODE REVIEW

File: component/remote/abs.go [hunk 1]
```diff
@@ -47,12 +46,7 @@ func (a *AbsApolloConfig) SyncWithNamespace(namespace string, appConfigFunc func
 		IsRetry: true,
 	}
 	if appConfig.SyncServerTimeout > 0 {
-		duration, err := time.ParseDuration(strconv.Itoa(appConfig.SyncServerTimeout) + "s")
-		if err != nil {
-			log.Errorf("parse sync server timeout %s fail, error:%v", appConfig.SyncServerTimeout, err)
-			return nil
-		}
-		c.Timeout = duration
+		c.Timeout = time.Duration(appConfigFunc().SyncServerTimeout) * time.Second
```
Comment (nobodyiam) [inline component/remote/abs.go:49]:
```suggestion
		c.Timeout = time.Duration(appConfig.SyncServerTimeout) * time.Second
```

File: component/serverlist/sync.go [hunk 2]
```diff
@@ -81,11 +80,7 @@ func SyncServerIPList(appConfigFunc func() config.AppConfig) (map[string]*config
 		Secret: appConfig.Secret,
 	}
 	if appConfigFunc().SyncServerTimeout > 0 {
-		duration, err := time.ParseDuration(strconv.Itoa(appConfigFunc().SyncServerTimeout) + "s")
-		if err != nil {
-			return nil, err
-		}
-		c.Timeout = duration
+		c.Timeout = time.Duration(appConfigFunc().SyncServerTimeout) * time.Second
```
Comment (nobodyiam) [inline component/serverlist/sync.go:83]:
```suggestion
		c.Timeout = time.Duration(appConfig.SyncServerTimeout) * time.Second
```

File: component/serverlist/sync.go [hunk 3]
```diff
@@ -81,11 +80,7 @@ func SyncServerIPList(appConfigFunc func() config.AppConfig) (map[string]*config
 		Secret: appConfig.Secret,
 	}
 	if appConfigFunc().SyncServerTimeout > 0 {
```
Comment (nobodyiam) [inline component/serverlist/sync.go:82]:
```suggestion
	if appConfig.SyncServerTimeout > 0 {
```

File: component/remote/abs.go (hunk 1 above)
Comment (nobodyiam) [reply component/remote/abs.go:49]: BTW, is it necessary to keep the error handle logic?

Comment (wzy9607) [reply component/remote/abs.go]: done

File: component/serverlist/sync.go (hunk 2 above)
Comment (wzy9607) [reply component/serverlist/sync.go]: done

File: component/serverlist/sync.go (hunk 3 above)
Comment (wzy9607) [reply component/serverlist/sync.go]: done

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**